<ui version="4.0" >
 <class>AkuViewOptionWidget</class>
 <widget class="QWidget" name="AkuViewOptionWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QLabel" name="viewlabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text" >
          <string>Current View:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox" >
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground" >
          <bool>false</bool>
         </property>
         <property name="sizeAdjustPolicy" >
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <property name="frame" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="iconBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title" >
        <string>Icon View</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>Icon Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="zoomOutButton" >
            <property name="text" >
             <string/>
            </property>
            <property name="autoRaise" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="iconSizeSlider" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="zoomInButton" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="autoRaise" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="treeBox" >
       <property name="font" >
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title" >
        <string>Tree View</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
